### PR TITLE
Feat(website): Setup website configuration according to criteria documentation

### PR DESCRIPTION
**Pull request by CJGutz**
closes: #1 

## Proposed changes
Adds a lot of initial package configuration for the project.

| Package | Configured with |
|--------|--------|
| Shadcn/ui | [Shadcn installation](https://ui.shadcn.com/docs/installation/next) |
| Typedoc | [Typedoc installation](https://typedoc.org/guides/installation/) |
| Continuous integration | Earlier projects (Ei) and [syntax docs](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions) |
| Cypress | [Next cypress](https://nextjs.org/docs/pages/building-your-application/testing/cypress) |
| Tailwind prettier | [Installation](https://tailwindcss.com/blog/automatic-class-sorting-with-prettier) | 

Made changes to the folder structure to follow nextjs' best practices with folder router in src/app/(pages)/
Other common parts to the project are in app directory. For example utilities and common components.

## Pull request checklist

Please check if your PR fulfills the following requirements:

- [x] Tests for the changes have been added
- [x] Documentation have been written

### CODE REVIEW

File: src/app/(pages)/layout.tsx
Comment (CJGutz) [inline src/app/(pages)/layout.tsx:1]: Only `variable` font and header/nav is added to this file

File: src/app/globals.css
Comment (CJGutz) [inline src/app/globals.css:1]: Deleted by shadcn

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
+name: Continous Integration
+
+on: push
```
Comment (thomsen85) [inline .github/workflows/ci.yaml:3]: Switch this to on pull request and push on main

**Review by thomsen85 — APPROVED** [review]
LGTM!

**Pull request merged**